robot-name/robot-name_test.spec.js

```diff
@@ -12,9 +12,17 @@ describe("Robot", function() {
   });
 
   it("different robots have different names", function() {
-    var robotOne = new Robot();
-    var robotTwo = new Robot();
-    expect(robotOne.name).not.toEqual(robotTwo.name);
+    var usedNames= {};
+
+    var i = 0,
+        max = 10000;
+
+    for (i; i < max; i++) {
```

Comment:
Commit 8a68287e48b61f7c56ca3e46632c99fed1d803ea follows rchavarria's suggestion.

I'm still new to people caring about how I git. Should I have somehow changed old commits instead of just adding new ones?